repository/flink/docs/demo/tests/flink-demo/00-assert.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kudo-controller-manager-0
+  namespace: kudo-system
+status:
+  phase: Running
+
+# confirms that kudo is running
```

Comment:
Why is it necessary to check this here?

Reply:
fail fast and portability
frankly if kudo is running this is a quick sub second check...  what are the concerns?

Reply:
it's not a pattern we should promote: this is something that is covered `beforeAll` tests (by using `kudo init --wait`) option. I guess you could see it as very defensive programming but frankly, this shouldn't be necessary